index.js

```diff
@@ -71,7 +71,8 @@ module.exports = function ({
             stdout: process.stdout,
         });
 
-        cli.run().then((success) => {
+        cli.run().then((result) => {
+            const success = typeof(isSucceresultssNew) === 'object' ? result.success : result
```

Comment:
@maciejmaciejewski is this a typo or I'm missing something? `isSucceresultssNew ` 👀

Reply:
Indeed, fixed it